R/load_fb.R

```diff
@@ -0,0 +1,107 @@
+#' Load match results
+#'
+#' Loading version of \code{get_match_results}
+#' Returns the game results for a given league season(s) from FBref
+#'
+#' @param country the three character country code
+#' @param gender gender of competition, either "M" or "F"
+#' @param season_end_year the year(s) the season concludes
+#' @param tier the tier of the league, ie '1st' for the EPL or '2nd' for the Championship and so on
+#'
+#' @return returns a dataframe with the results of the competition, season and gender
+#'
+#' @importFrom magrittr %>%
+#' @importFrom rlang .data
+#' @importFrom utils read.csv
+#'
+#' @export
+#'
+#' @examples
+#' \donttest{
+#' df <- load_match_results(
+#' country = c("ITA"), gender = "M", season_end_year = 2021, tier = "1st"
+#' )
+#' # for results from English 1st div for men and women:
+#' df <- load_match_results(
+#' country = "ENG", gender = c("M", "F"), season_end_year = 2021, tier = "1st"
+#' )
+#'
+#' }
+
+load_match_results <- function(country, gender, season_end_year, tier) {
+  dat_urls <- paste0("https://github.com/JaseZiv/worldfootballR_data/blob/master/data/match_results/", country, "_match_results.rds?raw=true")
+
+  collect_date <- .file_reader("https://github.com/JaseZiv/worldfootballR_data/blob/master/data/match_results/scrape_time_match_results.rds?raw=true")
```

Comment:
i'm not sure i like this paradigm (having a separate data set for scrape times). what if we tried saving the timestamp as an attribute with the data set? e.g. https://github.com/nflverse/nflfastR-data/blob/91bb21f44596b474f2c4390e348c8ac44c24879c/R/save_stats.R#L13 I'm not totally sure if this is possible with RDS (i think it probably is)

Reply:
Have addressed this in [6fe808c](https://github.com/JaseZiv/worldfootballR/pull/130/commits/6fe808cd2e17acd83dd2864b5288d59f70c3c9f0)